core/backend/src/TxnManager.ts

```diff
@@ -268,10 +268,10 @@ export class TxnManager {
    */
   public readonly onAfterUndoRedo = new BeEvent<(isUndo: boolean) => void>();
 
-  /** Determine whether undo is possible, optionally permitting undoing txns from previous sessions.
-   * @param allowCrossSessions if true, allow undoing from previous sessions.
+  /** Determine whether undo is possible
+   * @@deprecated - use [[isUndoPossible]]
    */
-  public checkUndoPossible(allowCrossSessions?: boolean) { return this._nativeDb.isUndoPossible(allowCrossSessions); }
+  public checkUndoPossible() { return this._nativeDb.isUndoPossible(); }

```

Comment:
This breaks a `@public` API, did you consider just documenting that the arg is ignored?

Reply:
the worst thing would be for someone to be passing `true` and have it silently start working differently. Better to get a compile error. But, as I said in the commit message, I'm sure no one is.